src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+# Metrics
+
+This section describes how to enable metrics reporting on the Policy Server.
+
+We are going to use [Prometheus](https://prometheus.io/) to scrape metrics exposed by the Policy
+Server.
+
+## Install Prometheus
+
+We will use the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator),
+that allows us to intuitively define Prometheus' Targets.
+
+There are many ways to install and set up prometheus. For ease of deployment, we will use the
+Prometheus community helm chart.
+
+Let's add the helm repository from the Prometheus Community:
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+```
+
+Now, let's install the
+[`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+chart. This chart contains a collection of Kubernetes manifests, Grafana dashboards, and Prometheus
+rules.
+
+Let's create a `kube-prometheus-stack-values.yaml` file to configure the `kube-prometheus-stack`
+Helm chart values with the following contents:
+
+```yaml
+prometheus:
+  additionalServiceMonitors:
+    - name: kubewarden
+      selector:
+        matchLabels:
+          app: kubewarden-policy-server-default
```

Comment:
Unrelated with this PR, but still... Shouldn't we use a more generic name here? I'm thinking about when we will have the Kubewarden controller expose metrics too. I guess it will have to use the same label in order to be scraped by Prometheus.

Reply:
This label is added by the controller, we are just targeting it here. https://github.com/kubewarden/kubewarden-controller/blob/9d0d1a02817eb12f82fffd5ea4a792b15cd35baf/internal/pkg/admission/policy-server-service.go#L30.